tasks/HandMeThat.tex

```diff
@@ -0,0 +1,60 @@
+\section{Hand Me That [Party Host]}
+A guest at the party speaks English, but with only a limited vocabulary. The robot will assist them in obtaining things that they gesture for.
+
+%\subsection{Focus}
+%Joint attention is a well-studied and important task in Human-Robot Interaction. The goal of this task is to really challenge the teams to perform a hard HRI task.
+
+\subsection{Main Goal}
+Five collections of items are set out. The referee will choose a random item from each collection to gesture to or describe to the robot. The robot must tell the referee its name, point to, or touch it. The robot can prompt the user with requests like \textit{point to it} or \textit{describe it to me}.
+
+\noindent\textbf{Reward:} 2500pts\\
+
+
+% %% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+% Setup
+%
+% %% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\subsection{Setup}
+\begin{enumerate}
+	\item \textbf{Object Collections:} The test area should collections of objects on various surfaces (2-5). Difficulty of understanding the point gesture will be regulated by how far apart the objects in each collection are ($100cm - 1m$). Difficulty of describing the object is difficult to quantify, and should be considered random.
```

Comment:
"Test area should **have?** collections..."

It might be difficult to always have 5 surfaces setup over the course of one test slot. Maybe just have five objects ready and then arrange them. Also as far as I can see difficulty and distances do not play a role in scoring?